src/pudl/analysis/epacamd_eia.py

```diff
@@ -205,7 +193,7 @@ def _convert_global_id_to_composite_id(
     else:
         idx_col = idx_name
 
-    composite_key: pd.Series = reindexed.groupby("CAMD_PLANT_ID", as_index=False).apply(
+    composite_key: pd.Series = reindexed.groupby("plant_id_eia", as_index=False).apply(
```

Comment:
Shouldn't this be "plant_id_epa"? (This applies to all instances where "CAMD_PLANT_ID" was replaced with "plant_id_eia")

Reply:
Or "plant_id_camd" (not sure what the new naming convention is)

Reply:
Technically `CAMD_PLANT_ID` translates to `plant_id_epa` in our new version. However, the `plant_id_eia` field is a little more restrictive because there are some cases where `plant_id_epa` maps to more than one `plant_id_eia`. Given that `plant_id_eia` is the "more correct" plant designation, I figured any time we sort on plant we should be using `plant_id_eia`